.teamcity/pom.xml

```diff
@@ -101,4 +101,4 @@
       <scope>compile</scope>
     </dependency>
   </dependencies>
-</project>
```

Comment:
https://stackoverflow.com/questions/729692/why-should-text-files-end-with-a-newline